compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -1049,7 +1049,8 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             (args, None)
         };
 
-        let mut copied_constant_arguments = vec![];
+        // Keeps track of temporary allocas whose liftime need to be ended after the call.
```

Comment:
typo, also slightly confusing wording to me, and often comments like this are written in the imperative, so perhaps this could be more like
```suggestion
        // Keep track of temporary allocas with a lifetime that ends after this call
```

Reply:
The exact phrasing isn't too important tho' and I don't necessarily insist on the imperative, please feel free to revise as you see fit.